### ISSUE TITLE
Unable to rsync if ssh root-login has been disabled

### ISSUE THREAD

**Issue by christopherraa** [smerrill/vagrant-gatling-rsync#32]
Hi,

Thanks for making a great plugin. We have however run into a small issue when trying to run duplicates of our prod-setup with Vagrant. Please see devopsgroup-io/vagrant-hostmanager#210 for a detailed explaination, as more or less the same issue exists there.

Would it be possible to use some other mechanism for authentication once provisioning is done and root-login has been disabled?

**Comment by smerrill** [comment]
Christopher,

Ultimately, I'm just calling to the `rsync` system already in Vagrant core and I don't really want to change that if I can help it, so it might be worth opening an issue on Vagrant core.

**Comment by christopherraa** [comment]
Thank you for the feedback. I'll see if they have any ideas.
